### PR TITLE
[T3A4][F09-A2]Xu Chen

### PR DESCRIPTION
**Pull request by githubxxcc**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/Main.java [hunk 1]
```diff
@@ -82,7 +82,7 @@ private void runCommandLoopUntilExitCommand() {
         do {
             String userCommandText = ui.getUserCommand();
             command = new Parser().parseCommand(userCommandText);
-            CommandResult result = executeCommand(command);
+            CommandResult result = executeCommand(command, storage);
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/Main.java]: It's unrelated to put `storage` as `executeCommand` param.

File: src/seedu/addressbook/Main.java [hunk 2]
```diff
@@ -103,8 +103,10 @@ private void recordResult(CommandResult result) {
      * @param command user command
      * @return result of the command
      */
-    private CommandResult executeCommand(Command command)  {
+    private CommandResult executeCommand(Command command, StorageFile storage)  {
         try {
+        	//check the address book before executing the command.
+        	this.addressBook=storage.load();
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/Main.java]: The task here asking you to use Exception to handle missing file when you call save method. Please check how other student complete this task, e.g. #591.

**Comment by jeffryhartanto** [comment]
Some comments added. Please close the PR after reading comments.
Pay attention to the commit before submitting your PR, since commit in this PR is mixed with commit in other task.

File: src/seedu/addressbook/Main.java (hunk 2 above)
Comment (githubxxcc) [inline src/seedu/addressbook/Main.java]: Sure. Thanks!

**Pull request closed without merging**